doc/en/how-to/usage.rst

```diff
@@ -201,28 +201,8 @@ hook was invoked:
 .. code-block:: pytest
 
     $ python myinvoke.py
-    .FEsxX.                                                              [100%]*** test run reporting finishing
+    *** test run reporting finishing
 
-    ================================== ERRORS ==================================
```

Comment:
Strange, why was this block removed? 🤔

Reply:
Looking at it in the context of the text above, I think the old output was wrong (other files leaking in somehow?) and the new one seems correct, no?